msprime/simulations.py

```diff
@@ -1944,11 +1981,32 @@ def __init__(self, reference_size=1, alpha=1.5, truncation_point=None):
 
 
 class DiracCoalescent(ParametricSimulationModel):
-    # TODO document
+    """
+    A diploid Xi-coalescent with up to four simultaneous multiple mergers and
+    crossover recombination. The string ``"dirac"`` can be used to refer to this
+    model.
+
+    The Dirac-Xi-coalescent is an implementation of the model of
+    Blath et al. (2013), see
+    https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3527250/.
+    The simulation proceeds very similarly to the standard coalescent.
+    In addition to all the possible events therein, there are potential
+    multiple merger events at fixed rate `c > 0`. Each lineage participates in
+    each multiple merger event independently with probability `0 < psi <= 1`,
+    all participating lineages are randomly split into four groups,
+    corresponding to the four parental chromosomes present in a diploid,
+    bi-parental reproduction event, and the lineages within each group merge
+    simultaneously.
+
+    Note that the Dirac-Xi-coalescent is obtained as the scaling limit of
+    Moran models, rather than Wright-Fisher models. As a consequence, one
+    unit of coalescent time is proportional to `Ne^2` generations, rather
+    than `Ne` generations as in the standard coalescent. Recombination and
+    other rates are still scaled by `Ne`.
```

Comment:
This is going to confuse people (certainly confuses me!). Perhaps we should point people to the tutorial to show an example of why this is different and what to expect?

This probably warrants a formal warning as well. If you do this
```
.. warning:
    Note that the Dirac-Xi-coalescent is obtained as the scaling limit of
￼    Moran models, rather than Wright-Fisher models. As a consequence, one
￼    unit of coalescent time is proportional to `Ne^2` generations, rather
￼    than `Ne` generations as in the standard coalescent. Recombination and
￼    other rates are still scaled by `Ne`.

```
it should render it in a warning box.